css/extra-styles.css

```diff
@@ -31,7 +31,7 @@ div#map_canvas {
 have to add some extra css to the mapcontainer div */
 div#map_legend {
```

Comment:
map_legend flows down past the map itself, leading to some visual ocnfusion due to lack of **padding** o nthe left. Might have been good to add that here.

Reply:
Notice also the height of hte infowindows and the laco of a scrollbar. Can you fix those?